UT-IfcCourse/UT-IfcCourse.ifc

```diff
@@ -0,0 +1,56 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition[Ifc4x3NotAssigned]'), '2.1');
+FILE_NAME ('UT-IfcCourse.ifc', '2021-02-17T09:25:56', (), ('AKG Software Consulting GmbH'), $, $, $);
+FILE_SCHEMA (('IFC4X3_RC2'));
+ENDSEC;
+DATA;
+#1 = IFCORGANIZATION('AKG Software Consulting GmbH', 'AKG Software Consulting GmbH', $, $, $);
+#2 = IFCAPPLICATION(#1, 'Build58', 'VESTRA INFRAVISION', 'VESTRA INFRAVISION');
+#3 = IFCPERSON($, 'VESTRA', 'VESTRA', $, $, $, $, $);
+#4 = IFCPERSONANDORGANIZATION(#3, #1, $);
+#5 = IFCOWNERHISTORY(#4, #2, $, $, $, $, $, 1613550356);
+#6 = IFCDIMENSIONALEXPONENTS(0, 0, 0, 0, 0, 0, 0);
+#7 = IFCUNITASSIGNMENT((#8, #9, #10));
+#8 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#9 = IFCSIUNIT(*, .TIMEUNIT., $, .SECOND.);
+#10 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#11 = IFCAXIS2PLACEMENT3D(#12, $, $);
+#12 = IFCCARTESIANPOINT((0., 0., 0.));
+#13 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #11, $);
+#14 = IFCPROJECTEDCRS('EPSG:25832', 'ETRS89_UTM32', 'EPSG:25832', 'EPSG:25832', 'UTM', '32', #8);
+#15 = IFCMAPCONVERSION(#13, #14, 0., 0., 0., $, $, $);
+#16 = IFCLOCALPLACEMENT($, #17);
+#17 = IFCAXIS2PLACEMENT3D(#18, $, $);
+#18 = IFCCARTESIANPOINT((51.888, -7.639, 0.));
+#19 = IFCPROJECT('1W7lFkDRwMWKYdP6AbIlsD', #5, 'Projekt: UT-IfcCourse', 'Projekt: UT-IfcCourse', $, $, $, (#13), #7);
+#20 = IFCSITE('1d1nf$UMdHWK5jSGoE5PmO', #5, 'VESTRA', 'VESTRA', $, #16, $, $, $, $, $, $, IFCLABEL('VESTRA'), $);

```

Comment:
Selects or enums require a type before the value. Otherwise, the simple defined type is not serialized in the file. For IfcLabel that means the following:
IFCSITE('1d1nf$UMdHWK5jSGoE5PmO', #5, 'VESTRA', 'VESTRA', $, #16, $, $, $, $, $, $, 'VESTRA', $);